CHANGES.rst

```diff
@@ -303,6 +303,9 @@ astropy.convolution
 astropy.coordinates
 ^^^^^^^^^^^^^^^^^^^
 
+- Changed the timescale in ICRS to CIRS from 'tdb' to 'tt' conversion and vice-versa,
```

Comment:
@pllim if I intend to backport this to 2.0.x this changelog entry should be in another place, am I right?

@shreyasbapat, could you please move it?

Reply:
Definitely!

Reply:
Hehe, can you please tell me where?

Reply:
In 2.0.x's Changelog? Which I suppose not!

Reply:
Gotcha!